docs/api.md

```diff
@@ -254,3 +254,16 @@ the `requestId` in the `monetizationstart` browser event.
 ```http
 Web-Monetization-Id: dcd479ad-7d8d-4210-956a-13c14b8c67eb
 ```
+
+## Iframes
+You can monetize an iframe by adding `monetization` to the iframe's `allow` attribute. For example:
+
+```html
+  <iframe src = "/wm/example.htm" width = "600" height = "800" allow="monetization">
+```
+
+The iframe **must** have the meta tag in its head. In the example above, `example.htm` contains the Web Monetization `meta` tag.
```

Comment:
I would clarify that it must have the meta tag in order to monetize. No error will occur just from embedding an iframe with allow='monetization' and not having it be monetized

Reply:
Updated. Can you recheck please?